core/src/main/scala/org/apache/spark/memory/StaticMemoryManager.scala

```diff
@@ -104,6 +104,8 @@ private[spark] class StaticMemoryManager(
 
 private[spark] object StaticMemoryManager {
 
+  private val MIN_MEMORY_BYTES = 450 * 1024 * 1024
```

Comment:
Rather than bother with a trait, just make it a constant in the `MemoryManger` companion object?